settings.gradle

```diff
@@ -12,3 +14,15 @@ include 'dataline-server'
 include 'dataline-webapp'
 include 'dataline-workers'
 include 'dataline-tests'
+
+// include all integration subprojects
+def prefix = rootProject.projectDir.absolutePath
+def dir = new File(prefix + "/dataline-integrations")
```

Comment:
why not
```
def dir = rootProject.file("dataline-integrations")
```

Reply:
`rootProject` is a `ProjectDescriptor` not a `Project` and doesn't have a `file` method.

I think what I changed it to now is cleaner than it was originally though.